ibis/backends/tests/test_numeric.py

```diff
@@ -254,7 +255,7 @@ def test_numeric_literal(con, backend, expr, expected_types):
                 "postgres": decimal.Decimal("1.1"),
                 "pandas": decimal.Decimal("1.1"),
                 "pyspark": decimal.Decimal("1.1"),
-                "mysql": 1.1,
+                "mysql": decimal.Decimal("1"),
```

Comment:
This isn't `"1.1"` due to the behavior of MySQL's default decimal parameters. See the sqlglot type mapper for MySQL for details.

Reply:
The ibis datatype of 
```py
ibis.literal(decimal.Decimal("1.1"), type=dt.decimal).type()
```
is
```
Decimal(precision=None, scale=None, nullable=True)
```

so the literal is casted to `DECIMAL` and MySQL decides about scale and precision. 

We should have another test case where we test a decimal literal with explicit scale and precision, and we expect uniform behaviour across all the backends.

Reply:
Created a follow-up https://github.com/ibis-project/ibis/issues/7930